Task01/NarcissisticNumbers.py

```diff
@@ -0,0 +1,33 @@
+import os,sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+def main():
+    print_NarcissisticNumbers(1,1000)
+
+def print_NarcissisticNumbers(minNumber,maxNumber):
+    input_list = list(range(minNumber,maxNumber,1))
+    for input_Item in input_list:
+        if input_Item == narcissistic_function(input_Item):
+            print(input_Item, end =" ")
```

Comment:
```suggestion
            print(input_Item, end=" ")
```
keyword parameters указывают без пробелов